translations/frontend/it.json

```diff
@@ -4241,7 +4241,7 @@
                             "gas": "Gas",
                             "go_to_energy_dashboard": "Vai alla plancia energia",
                             "grid": "Rete",
-                            "home": "Home",
+                            "home": "Casa",
```

Comment:
Translations are not managed via GitHub, if you want to contribute and help with Translating Home Assistant, please check out our documentation on how to do this:

<https://developers.home-assistant.io/docs/translations>